### PR TITLE
fix: update kubeone.yaml

### PR DESCRIPTION
**Pull request by Javlopez**
- the fix resolves problem with CCM/CNI
- also added automatic installation of kubeone

### CODE REVIEW

File: kubernetes-cluster-kubeone/terraform/run.sh [hunk 1]
```diff
@@ -29,6 +29,21 @@ else
...
+  echo "Installing Kubeone..."
+  curl -sfL https://get.kubeone.io | sh
+  if ! command -v kubeone &> /dev/null; then
+    echo "unable to install kubeone please go to https://docs.kubermatic.com/kubeone/v1.7/tutorials/creating-clusters/ and install it"
+    exit 1
+  fi
+fi
+
+# remove kubeone folder
+find . -type d -name "kubeone_*" -exec rm -rf {} +
```
Comment (zalsader) [inline kubernetes-cluster-kubeone/terraform/run.sh]: Do we always want to delete this file or only under certain circumstances?

File: kubernetes-cluster-kubeone/terraform/kubeone.yaml [hunk 2]
```diff
@@ -4,15 +4,4 @@ versions:
...
-    external: {}
-
-addons:
-  enable: true
-  addons:
-    - name: calico-vxlan
-      params:
-        MTU: "0" # auto-detect MTU
\ No newline at end of file
+  external: true
```
Comment (zalsader) [inline kubernetes-cluster-kubeone/terraform/kubeone.yaml]: Add a new line at the end.

**Review by zalsader — APPROVED** [review]
Looks good! Does it fully work now?

File: kubernetes-cluster-kubeone/terraform/run.sh (hunk 1 above)
Comment (Javlopez) [reply kubernetes-cluster-kubeone/terraform/run.sh]: only after install kubeone, I already moved this code to just run once kubeone was installed

**Comment by Javlopez** [comment]
yes it is working fine now @zalsader

File: kubernetes-cluster-kubeone/terraform/run.sh [hunk 3]
```diff
@@ -29,6 +29,20 @@ else
     echo -e "Terraform initialized: \033[32m✔️\033[0m" 
 fi
 
+# check if kubeone is installed
```
Comment (zalsader) [inline kubernetes-cluster-kubeone/terraform/run.sh:32]: Do we want to use one specific version?

**Pull request merged**